src/sentry/migrations/0067_remove_tracked_superusers.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-04-20 20:03
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def remove_tracked_superuser_views(apps, schema_editor):
+    """
+    We recently added code to only track alert views of people in the org + member of associated alert projects.
+    This migration removes all of the views we've tracked before adding this change (i.e. superuser views of orgs we're not a part of).
+    """
+    IncidentSeen = apps.get_model("sentry", "IncidentSeen")
+    IncidentProject = apps.get_model("sentry", "IncidentProject")
+    tracked_views = IncidentSeen.objects.all()
+    for tracked_view in tracked_views:
+        tracked_user = tracked_view.user
+        tracked_incident = tracked_view.incident
+        is_org_member = tracked_incident.organization.has_access(tracked_user)
+        is_project_member = False
+        if is_org_member:
+            for incident_project in IncidentProject.objects.filter(incident=tracked_incident).select_related(
+                "project"
+            ):
+                if incident_project.project.member_set.filter(user=tracked_user).exists():
+                    is_project_member = True
+                    break
```

Comment:
I think we should probably skip this check. There are manager/owner level permissions in an org, and they're allowed to see things related to projects they're not explicit members of. `organization.has_access` checks that they're a member of the org, which should be good enough to remove only superusers.